gnosis/cowswap
# [Claim] Rejected/failed/errors in claim flow

# Summary

#### Screenshots
_Approving_
<img width="814" alt="Screenshot 2022-01-17 at 11 14 20" src="https://user-images.githubusercontent.com/21335563/149781704-cc4c8211-b524-4e5c-97f4-5b023d29414b.png">

_Airdrop Claim (first page)_
![image](https://user-images.githubusercontent.com/21335563/149781898-7c062a1b-75cb-4c7f-b7d6-1362c6517e46.png)

## To Test
AIRDROP
1. using an account with airdrop and investments: 0xB1B98E968E9D7c6Ee1aa3496A753ae8922FC4683
2. first only leave airdrop selected (the preselected one)
3. hit claim
4. reject wallet tx
5. check error is as my screenshot shows
INVESTMENT
1. using an account with airdrop and investments: 0xB1B98E968E9D7c6Ee1aa3496A753ae8922FC4683
2. select the investment option (unselected one)
3. go thru the screens until on the investment approve screen
4. since this acct already has approves, click the "invest max" button (it is test coded to reject)
5. check error is as my screenshot shows for investment
